build-image/webif.bash

```diff
@@ -31,6 +31,7 @@ if [ "$1" = "start" ]; then
   (cd /tmp/webif || exit 1; python3 -m http.server $port > /dev/null 2>&1 &)
 fi
 
+# changed to python3 due to issues in automated testing
```

Comment:
...what you explained was not related to automated testing, it was about the webif, right?

Reply:
yes it was the webif to fail, but the occasion this showed up was during automated testing